mkdocs.yml

```diff
@@ -73,89 +62,98 @@ nav:
+           - Use Wallets for Key Management: Using-Pantheon/Account-Management.md
+           - Create and Send Transactions: Using-Pantheon/Transactions/Transactions.md
+           - Create and Send Private Transactions: Privacy/How-To/Creating-Sending-Private-Transactions.md
+           - Include Revert Reason in Transaction Receipts: Using-Pantheon/Transactions/Revert-Reason.md
+        - Deploy for Production:
+          - Use Ansible to Deploy Pantheon: Deploying-Pantheon/Ansible.md
+          - Use Prometheus for Monitoring: Monitoring/Monitoring-Performance.md
+          - Use Alethio Lite Explorer: Monitoring/Alethio/Lite-Block-Explorer.md
+          - Use Alethio EthStats Lite Network Monitor: Monitoring/Alethio/Lite-Network-Monitor.md
+          - Configure High Availability of APIs: Deploying-Pantheon/High-Availability.md
+          - Deploy Permissioning Management Dapp: Permissions/Onchain-Permissioning/Production.md
+        - Find and Connect to Peers:
+          - Start a Bootnode: Configuring-Pantheon/Networking/Bootnodes.md
+          - Configure Ports for Access: Configuring-Pantheon/Networking/Configuring-Ports.md
+          - Manage Peers: Configuring-Pantheon/Networking/Managing-Peers.md
+          - Use UPnP: Configuring-Pantheon/Networking/Using-UPnP.md
+        - Develop on Pantheon:
+          - Pass JVM Options: Configuring-Pantheon/Passing-JVM-Options.md
+          - Use Truffle with Pantheon: Using-Pantheon/Truffle.md
+        - Mine:
+          - Configure Mining: Using-Pantheon/Mining.md
+        - Troubleshoot:
+          - Configure Logging: Monitoring/Logging.md
+          - Solve Common Problems: Troubleshooting/Troubleshooting.md
+    - Explanation:
+        - Architecture: Architecture/Overview.md
+        - Consensus Protocols:
+          - Overview: Consensus-Protocols/Overview-Consensus.md
+          - Comparing PoA Consensus Protocols: Consensus-Protocols/Comparing-PoA.md
+        - Privacy:
+          - Overview: Privacy/Explanation/Privacy-Overview.md
           - Privacy Groups: Privacy/Explanation/Privacy-Groups.md
           - Processing Private Transactions: Privacy/Explanation/Private-Transaction-Processing.md
-    - Permissioning:
-        - Overview: Permissions/Permissioning-Overview.md
-        - Local Permissioning: Permissions/Local-Permissioning.md
-        - Onchain Permissioning:
-            - Overview: Permissions/Onchain-Permissioning/Onchain-Permissioning.md
-            - Getting Started with Onchain Permissioning: Permissions/Onchain-Permissioning/Getting-Started-Onchain-Permissioning.md
-            - Deploying Management Dapp for Production: Permissions/Onchain-Permissioning/Production.md
-            - Updating Whitelists: Permissions/Onchain-Permissioning/Updating-Whitelists.md
-    - Using Pantheon:
+        - Permissioning:
+          - Overview: Permissions/Permissioning-Overview.md
+          - Onchain Permissioning: Permissions/Onchain-Permissioning/Onchain-Permissioning.md
```

Comment:
Why not having local perm here too?

Reply:
The permissioning topics didn't break out that well.  I'll have to do some rejigging on them in a separate issue.

Reply:
https://pegasys1.atlassian.net/browse/PAN-3094